packages/astro/src/i18n/vite-plugin-i18n.ts

```diff
@@ -0,0 +1,26 @@
+import * as vite from 'vite';
+import type { AstroSettings } from '../@types/astro.js';
+import type { Logger } from '../core/logger/core.js';
+
+const virtualModuleId = 'astro:i18n';
+const resolvedVirtualModuleId = '\0' + virtualModuleId;
+
+type AstroInternalization = {
+	settings: AstroSettings;
+	logger: Logger;
+};
+
+export default function astroInternalization({ settings }: AstroInternalization): vite.Plugin {
```

Comment:
```suggestion
type AstroInternationalization = {
	settings: AstroSettings;
	logger: Logger;
};

export default function astroInternationalization({ settings }: AstroInternationalization): vite.Plugin {
```

I wouldn't mind if it's shorten as i18n here too 🙈

Reply:
Will change that, cheers